### PR TITLE
CUMULUS-3841: Increase fetchRules page size to improve performance.

### PR DESCRIPTION
**Pull request by paulpilone**
**Summary:** Summary of changes

Addresses [CUMULUS-3841 Increase fetchRules page limit size to improve overall query performance: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3841)

## Changes

* User contributed PR for issue: https://github.com/nasa/cumulus/issues/3772
* Original PR: https://github.com/nasa/cumulus/pull/3773

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by paulpilone** [comment]
Addresses #3772

File: packages/api/lib/rulesHelpers.js
Comment (etcart) [inline packages/api/lib/rulesHelpers.js:1]: can we add to our unit tests a validation of this?

Comment (paulpilone) [reply packages/api/lib/rulesHelpers.js:1]: Updated failing test and added a small one that validates the new argument. Fixed a small doc typo too.

Comment (etcart) [reply packages/api/lib/rulesHelpers.js:1]: looks awesome!

**Review by etcart — APPROVED** [review]

**Pull request merged**